crates/bevy_picking/src/pointer.rs

```diff
@@ -29,41 +27,69 @@ use crate::backend::HitData;
 #[derive(Debug, Default, Clone, Copy, Eq, PartialEq, Hash, Component, Reflect)]
 #[require(PointerLocation, PointerPress, PointerInteraction)]
 #[reflect(Component, Default, Debug, Hash, PartialEq)]
-pub enum PointerId {
-    /// The mouse pointer.
-    #[default]
-    Mouse,
-    /// A touch input, usually numbered by window touch events from `winit`.
-    Touch(u64),
-    /// A custom, uniquely identified pointer. Useful for mocking inputs or implementing a software
-    /// controlled cursor.
-    #[reflect(ignore)]
-    Custom(Uuid),
+pub struct PointerId {
+    /// A stable identifier for a specific pointer.
+    ///
+    /// This is typically provided by winit,
+    /// and is unique for each touch input or mouse.
+    pub id: u64,
+    /// The type of hardware device that created this pointer.
+    pub kind: PointerKind,
 }
 
 impl PointerId {
+    /// The [`PointerId`] of the mouse.
+    ///
+    /// Currently, only one mouse pointer is supported.
+    pub const MOUSE: Self = Self {
+        id: 0,
+        kind: PointerKind::Mouse,
+    };
+
     /// Returns true if the pointer is a touch input.
     pub fn is_touch(&self) -> bool {
```

Comment:
I've left these utility methods alone to ease migration, but I'm not convinced they're particularly valuable now.